### ISSUE TITLE
FTB Interactions crashes as soon as i get into the game

### ISSUE THREAD

**Issue by grandogarndo** [FTBTeam/FTB-Interactions#829]
## Modpack Version: 
1.9.1

## Bug Report:
Whenever I get into a created world, the game crashes and sends me back to the FTB launcher.

## Expected Behavior/Suggestions:
It should let me play, there is enough ram dedicated

## Is It Repeatable? Steps to Reproduce: 
1.Log on to FTB Legacy launcher and load 1.9.1 FTB Interactions
2. Join a world and see if it crashes

## Link to Log or Crash File Paste**: 
https://paste.feed-the-beast.com/view/204d15cd

## Mod/s Affected: 
I read that justenoughdimensions might be the problem

**Comment by Saereth** [comment]
This doesnt actually show a crash in the logfile, did you post the wrong one?

**Comment by grandogarndo** [comment]
Oh, sorry, I forgot about this a long time ago, I really don't know if that is the right one. I do remember that I wasn't able to join no matter how much ram and tabs I closed. I have no idea if that is the right crash log, but whenever I tried to make or join a world I could play for a second then boom I crashed. I don't really have time to recreate it now either, but I can post another crash this weekend.
